File: reference.net/db4o-reference-chapters/src/Db4objects.Db4odoc/Refactoring/Pilot.cs
/* Copyright (C) 2004 - 2007 db4objects Inc. http://www.db4o.com */
namespace Db4objects.Db4odoc.Refactoring
{
	public class Pilot
	{
		private string _name;
		
		public Pilot(string name) 
		{
			this._name=name;
		}

		public string Name 
		{
			get 
			{
				return _name;
			}
			set 
			{
				_name = value;
			}
		}
		override public string ToString() 
		{
			return _name;
		}
	}
}

File: reference/src/com/db4odoc/enums/Pilot.java
/* Copyright (C) 2004 - 2007 db4objects Inc. http://www.db4o.com */

package com.db4odoc.enums;

public class Pilot {
	private String name;
    private Qualification qualification;

    public Pilot(String name,Qualification qualification) {
        this.name=name;
        this.qualification=qualification;
    }

    public Qualification getQualification() {
        return qualification;
    }

    public String getName() {
        return name;
    }

    public String toString() {
        return name+"/"+qualification;
    }
}
